components/utile.css

```diff
@@ -1,3 +1,217 @@
+.head-1 {
+  font-size: 48px;
+  line-height: 64px;
+}
+
+.head-2 {
+  font-size: 32px;
+  line-height: 40px;
+}
+
+.head-3 {
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.head-4 {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.head-5 {
+  font-size: 8px;
+  line-height: 16px;
+}
+
+.text-1 {
+  font-size: 48px;
+  line-height: 64px;
+}
+
+.text-2 {
+  font-size: 32px;
+  line-height: 40px;
+}
+
+.text-3 {
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.text-4 {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.text-5 {
+  font-size: 8px;
+  line-height: 16px;
+}
+
+.font-weight-light {
+  font-weight: 300;
+}
+
+.font-weight-regular {
+  font-weight: 400;
+}
+
+.font-weight-semibold {
+  font-weight: 600;
+}
+
+.font-weight-bold {
+  font-weight: 700;
+}
+
+.font-weight-black {
+  font-weight: 900;
+}
+
+.margin-xxs {
+  margin: 0.25rem;
+}
+
+.margin-xs {
+  margin: 0.5rem;
+}
+
+.margin-sm {
+  margin: 1rem;
+}
+
+.margin-md {
+  margin: 1.5rem;
+}
+
+.margin-lg {
+  margin: 2rem;
+}
+
+.margin-xl {
+  margin: 2.5rem;
+}
+
+.margin-xxl {
+  margin: 3rem;
+}
+
+.padding-xxs {
+  padding: 0.25rem;
```

Comment:
for any value less than 10px, you should give in px

Reply:
okay